lib/trento_web/controllers/fallback_controller.ex

```diff
@@ -17,6 +17,13 @@ defmodule TrentoWeb.FallbackController do
     |> render(:"404", reason: "SUSE Manager settings not configured.")
   end
 
+  def call(conn, {:error, :suma_authentication_error}) do
+    conn
+    |> put_status(:forbidden)
+    |> put_view(ErrorView)
+    |> render(:"403", reason: "SUSE Manager authentication error.")
+  end
+
```

Comment:
Since we now rely on 403 for our own permissions system it's better to switch this to 404 or 422, no big deal